_data/components/otableData/03tablePaginator.yml

```diff
@@ -9,7 +9,14 @@ attributes: [
   default : "10",
   required : "",
   description : "Number of items to display on a page. By default set to 10"
-}
+  },
+  {
+  name: "page-size-options", 
+  type: "array",
+  default : "[10, 25, 50, 100]",
```

Comment:
This parameter should follow the ';' convention.
All parameters in *ontimize-web-ngx* that are suitable to receive an array as value should receive a string with the values separated by `;`.
The usage of this parameter would be `page-size-options="20;50;100"` and the value will be converted into an array fy the framework.